ova/Libraries/wazuh_functions.sh

```diff
@@ -66,7 +69,7 @@ delete_logs(){
     systemctl stop wazuh-manager
     systemctl stop wazuh-api
 
-    find ./ -type f \( -iname \*.log -o -iname \*.json \) -exec : > {} \;
+    find ${DIRECTORY}/logs -type f \( -iname \*.log -o -iname \*.json \) -exec truncate -s 0 `echo {}` \;
```

Comment:
This can be simplified if you remove the `echo` from it. ```truncate -s 0 `echo {}` ``` to ```truncate -s 0 {}```

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/246/commits/b572065aad0e2fe15af945fbc58eb3428fcc5528